harmoni-data/src/main/java/org/harmoniapp/harmonidata/entities/Language.java

```diff
@@ -21,6 +22,8 @@ public class Language {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @Column(unique = true)
+    @NotEmpty(message = "Name is required")
     private String name;
 
     private String code;
```

Comment:
Code can be empty. Should it be?

Reply:
At this point, when adding an entity from LanguageService, the code field is not required, which means it can be null